deploy/cr.yaml

```diff
@@ -7,6 +7,11 @@ metadata:
 #  - percona.com/delete-ssl
 spec:
   crVersion: 2.6.0
+#  metadata:
+#    annotations:
+#      test-annotation: value
```

Comment:
Do we want to have a `test` prefix here in the CR? Maybe something like `example-annotation` or `annotation-key`. Test feels like a leftover from some testing.

Reply:
@pooknull wdyt about this?